### ISSUE TITLE
Cards would sometimes not display (all) installed creations

### ISSUE THREAD

**Issue by kyrie25** [spicetify/spicetify-marketplace#257]
### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.88.612.gcc529952
- Spicetify version: 2.10.1
```

### 📝 Description

Installed tab would sometimes not display (some) installed cards, regardless of the types. It can be fixed by reloading Spotify, but this only works roughly half of the time. One thing should be noted that the 2nd installed card of the same type appears consistently, but I have yet to understand why.

Issue have been replicated:
- On a fresh install of Marketplace
- After resetting Marketplace

### 📸 Screenshots

![Spotify_rdflAFsydJ](https://user-images.githubusercontent.com/77577746/175765850-66f58bd5-b11d-4914-8c86-95eb509581eb.gif)

![Spotify_etRqVU4J7V](https://user-images.githubusercontent.com/77577746/175765807-aa4b1ff4-992a-45ec-9cef-31b9bf66af34.gif)

**Comment by kyrie25** [comment]
Apparently it is looking for the wrong `localStorage` key?
![image](https://user-images.githubusercontent.com/77577746/175766476-c724bd02-db26-4d63-95cd-6db7700872f1.png)

**Comment by kyrie25** [comment]
Merged fix to #256
